buckwheat/tokenizer.py

```diff
@@ -266,6 +266,21 @@ def merge_nodes_for_lang(lang: str) -> Set[str]:
             class_types = TreeSitterParser.CLASSES[lang]
         return identifier_types | function_types | class_types
 
+    @staticmethod
+    def get_syntactic_tree_from_file(file: str, lang: str) -> Tuple[tree_sitter.Tree, bytes]:
```

Comment:
```suggestion
    def extract_ast(content: Union[str, bytes], lang: str) -> tree_sitter.Tree:
```
let's separate AST extraction from reading of file.

Reply:
If I'm separating ast extraction from reading fille I probably don't need Union[str, bytes]. Or should I read the file if the content is str?

Reply:
you should handle it properly - I mean reading and preparation of bytes/str